pkg/operators/register.go

```diff
@@ -123,6 +396,7 @@ func runJobController(opts standalone.BuildInfo, c *cli.Context, cfg *setting.Cf
 }
 
 func setupFromConfig(cfg *setting.Cfg) (controllerCfg *controllerConfig, err error) {
```

Comment:
Questions about settings: 

- Which is the convention we follow for the settings? how should they be structured? 
- Did I go too far with the configuration?